.github/workflows/ci.yml

```diff
@@ -72,10 +72,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - uses: arduino/setup-protoc@v1
-        with: { version: "3.x", repo-token: "${{ github.token }}" }
+      - uses: bufbuild/buf-setup-action@v1.26.1
+        with: {github_token: "${{ github.token }}"}
       - name: Generate protobuf code
-        run: protoc --js_out=/tmp spec/embedded_sass.proto
+        run: buf generate --output=/tmp --path=spec/embedded_sass.proto
```

Comment:
We should specify the input in `buf.work.yaml`, and I don't think there's a particular reason to generate to `/tmp` as opposed to the default location.

Reply:
done, and good point. For some reason I thought the CI would complain if there was uncommitted code 😅. That said, I added a `.gitignore` to prevent accidentally committing the generated code if we also accidentally run `buf generate` :)